tests/python/relay/test_op_grad_level10.py

```diff
@@ -24,5 +24,12 @@ def test_cross_entropy_grad():
     check_grad(relay.Function([x, y], relay.op.nn.cross_entropy(x, y)), eps=0.01, scale=0.1, mean=1)
 
 
+def test_checkpoint():
```

Comment:
I think there should probably also be a test of the tuple case to ensure we are trying all supported cases

Reply:
good call, caught a bug with the checkpoint identity compute when using tuples